templates/records/image-viewer.html

```diff
@@ -11,26 +11,6 @@
                 <div class="image-viewer__reference">
                     <h1>{{ images.has_other_pages }}<strong>{{ page.reference_number }}</strong> <span>- Image {{ index|add:'1' }} of {{ images_count }}</span></h1>
                 </div>
```

Comment:
Hi James,  

I noticed a couple issues  with the image viewer changes which may need to be fixed:

- When hitting full screen, the toolbar with Zoom In, Zoom Out, etc. goes back to the top. On my mobile, because "Exit Fullscreen" is now in the same place as the green "All Records" link, sometimes when I hit "Exit fullscreen" it also clicks the "All Records" link and  takes me back to  the page listing all the images. I'm guessing the toolbar moving is also an issue as the links are moving which was one of the accessibility issues.
-  If possible, I'm also wondering if the links could be given some more margin or padding as they are also quite close together when using mobile with touch screen.